packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -560,43 +560,41 @@ function ensureTargetGlobalExists(
 }
 
 /**
- * An object representing the changes at a particular snapshot version for the
- * given target. This is used to facilitate storing query changelogs in the
- * targetChanges object store.
+ * An object store to store the keys of changed documents. This is used to
+ * facilitate storing document changelogs in the Remote Document Cache.
  *
  * PORTING NOTE: This is used for change propagation during multi-tab syncing
  * and not needed on iOS and Android.
  */
-export class DbTargetChange {
+export class DbDocumentChanges {
```

Comment:
I'd name this to match DbRemoteDocuments since they're very tightly coupled... i.e. DbRemoteDocumentChanges

Reply:
Sounds fair.